File: 2475.py
def sqrt(n):
    return(n*n)    

a, b, c, d, e = map(int, input().split())
a = sqrt(a)
b = sqrt(b)
c = sqrt(c)
d = sqrt(d)
e = sqrt(e)
print((a+b+c+d+e) % 10)

# list = input().split(" ")
# sum = 0
# result = 0

# for i in list :
#     sum += (int(i)**2)
# result = sum % 10
# print(result)

File: bronze5/2475/2475.js
//문제: 2475
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
let input = fs.readFileSync(filePath).toString().trim().split("\n");

solution(input);
function solution(input) {
  const arr = input[0].split(" ").map(Number);
  let answer = 0;
  arr.forEach((e) => {
    answer += e * e;
  });
  console.log(answer % 10);
}
